test/e2e/generator/generate.go

```diff
@@ -99,12 +100,21 @@ func Generate(r *rand.Rand, opts Options) ([]e2e.Manifest, error) {
 			if opt["p2p"] == HybridP2PMode {
 				continue
 			}
+			if opt["txSize"].(int) != 2048 {
```

Comment:
Do we want this? Won't this basically skip all the other sizes we wanted?

Reply:
yeah, but only on single-node networks.